src/TheRestOfYourLife/hittable_list.h

```diff
@@ -43,20 +43,7 @@ class hittable_list : public hittable {
         return hit_anything;
     }
 
-    bool bounding_box(aabb& output_box) const override {
-        if (objects.empty()) return false;
-
-        aabb temp_box;
-        bool first_box = true;
-
-        for (const auto& object : objects) {
-            if (!object->bounding_box(temp_box)) return false;
-            output_box = first_box ? temp_box : aabb(output_box, temp_box);
-            first_box = false;
-        }
-
-        return true;
-    }
+    aabb bounding_box() const override { return bbox; }
```

Comment:
needs `add` function?

Reply:
_Yoiks_. Good catch. Also missing `aabb.h` include. I'll add a new finalization task to compare source from book to book (many files in books 2 & 3 are the same or obviously similar).